app/tests/test_generic_api.py

```diff
@@ -402,3 +402,80 @@ def test_head_cache_header(self):
                 self.assertAlmostEqual((expires - now).total_seconds(),
                                        timedelta(seconds=3600).total_seconds(),
                                        delta=2)
+
+
+class ApiCORSHeaderTestCase(StacBaseTestCase):
+
+    @classmethod
+    @mock_s3_asset_file
+    def setUpTestData(cls):
+        cls.factory = Factory()
+        cls.collection = cls.factory.create_collection_sample(
+            name='collection-1',
+            db_create=True,
+        )
+        cls.item = cls.factory.create_item_sample(
+            collection=cls.collection.model,
+            name='item-1',
+            db_create=True,
+        )
+        cls.asset = cls.factory.create_asset_sample(
+            item=cls.item.model,
+            name='asset-1',
+            db_create=True,
+        )
```

Comment:
Just as a side note, here the `name='collection|item|asset-1'` would not be required as it is the default value of the create sample.

Reply:
removed it